sympy/utilities/tests/test_pickling.py

```diff
@@ -97,7 +96,7 @@ def test_core_basic():
     for c in (Atom, Atom(),
               Basic, Basic(),
               # XXX: dynamically created types are not picklable
-              # ManagedProperties, ManagedProperties("test", (), {}),
+              # type, type("test", (), {}),
```

Comment:
Is this comment still neccesary? It made sense when it gave a reason why `ManagedProperties` was not being tested here, but now it might be more confusing than helpful. `object` is not being tested here either.

Reply:
Well dynamically created types are still not picklable. I'm not sure why that was relevant when this comment was added.

Reply:
This is checking if classes defined in sympy (a metaclass still is a class) are pickable, which indeed did not work for `ManagedProperties` and it would make sense to leave this comment here for the next person.

Reply:
Your comment reads almost like it has a missing "not" (would *not* make sense). I can leave or remove this comment from the code...

Reply:
If the class `ManagedProperties` exists, it makes sense to have the comment (so in the past it did make sense, this was an answer to the question why it was relevant when this comment was added)
If the class `ManagedProperties` does not exist, it does not make sense to have the comment

It does not make sense to put testing for `type` in this list and then comment it out and have a comment stating that it can't be pickled.
This function is testing SymPy classes, not python built-ins

Reply:
Okay, I've removed the comment.